bundles/ui/org.eclipse.smarthome.ui.classic/snippets/rollerblind.html

```diff
@@ -1,5 +1,5 @@
 <li>
-	<img src="../icon/%icon%.png" width=29 height=29 class="iFull" />
+	<img src="../icon/%icon%?format=%format%&state=%state%" width=29 height=29 class="iFull" />
```

Comment:
%icon% is always the category of an item, right? Should we change it to %category% to make it clear